west.yml

```diff
@@ -108,7 +108,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: f215460bec9869a2122059af28550792358a77e5
+      revision: pull/685/head
```

Comment:
Please update SHA and rebase: `ba6a4024db239413a2e097718b0777ce4c2a75d0`

Reply:
Done.